packages/api-page-builder/src/graphql/graphql/pages.gql.ts

```diff
@@ -1,11 +1,18 @@
-import { ListResponse, Response, ErrorResponse } from "@webiny/handler-graphql/responses";
+import {
+    ListResponse,
+    Response,
+    ErrorResponse,
+    NotFoundResponse
+} from "@webiny/handler-graphql/responses";
 import { GraphQLSchemaPlugin } from "@webiny/handler-graphql/types";
 import { Page, PbContext } from "~/types";
 import Error from "@webiny/error";
 import resolve from "./utils/resolve";
 import pageSettings from "./pages/pageSettings";
 import { fetchEmbed, findProvider } from "./pages/oEmbed";
 import lodashGet from "lodash/get";
+import { PageSecurityPermission } from "@webiny/api-page-builder/types";
+import checkBasePermissions from "@webiny/api-page-builder/graphql/crud/utils/checkBasePermissions";
```

Comment:
BTW @kedrzu, check these two imports. Use "~" when importing from package root.

Wouldn't mention this, but CI/CD threw an error related to these two imports. Using "~" will fix it.

Reply:
Also, `PageSecurityPermission` can be added to the import statement in line 8.